tests/integration/internal/receivermock/receiver_mock.go

```diff
@@ -0,0 +1,70 @@
+package receivermock
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+	"testing"
+
+	http_helper "github.com/gruntwork-io/terratest/modules/http-helper"
+)
+
+// A HTTP client for the receiver-mock API
+type ReceiverMockClient struct {
+	baseUrl   url.URL
+	tlsConfig tls.Config
+	t         *testing.T
```

Comment:
I'd prefer not to store this. Instead we could pass it as first param to exposed public functions (like in other "frameworks")